tools/clang-tidy-plugin/test/lit.cfg

```diff
@@ -21,8 +21,9 @@ check_clang_tidy += ' -std=c++17'
 # Note: backlash (`\`) in the path seems to cause it to not work properly
 # on Windows, therefore the relative path should start with `./` and use `/`
 # to delimit directories instead of using multiple arguments to `os.path.join`.
-cata_include = os.path.join( config.cata_source_dir, "./src" )
-test_include = os.path.join( config.cata_source_dir, "./tools/clang-tidy-plugin/test-include" )
+cata_include = os.path.join( config.cata_source_dir, "src" )
+cata_third_party_include = os.path.join( config.cata_source_dir, "src", "third-party" )
+test_include = os.path.join( config.cata_source_dir, "tools", "clang-tidy-plugin", "test-include" )
```

Comment:
See the comment above, you should keep the `./` and `/` for it to work on Windows.